generators/server/templates/src/main/docker/cassandra/scripts/_execute-cql.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+function log {
+    echo "[$(date)]: $*"
+}
+
+KEYSPACE_NAME=<%= baseName.toLowerCase() %>
+
+#usage checks
+if [ -z "$1" ]
+  then
+    echo "usage: ./execute-cql cqlFile.cql"
+    exit 1
+fi
+if [ -z "$CASSANDRA_CONTACT_POINT" ]
+  then
+    echo "CASSANDRA_CONTACT_POINT environment variable must be defined"
+    exit 1
+fi
+
+cqlFile=$1
+
+log "execute: " $cqlFile
+cqlsh -k $KEYSPACE_NAME -f $cqlFile $CASSANDRA_CONTACT_POINT
+
+if [ $? -ne 0 ]; then
+    log "fail to apply script " $filename
+    log "stop applying database changes"
+    exit 1
+fi
+
+exit 0
```

Comment:
I think it is the `exit 0` I have added by mistake when I've extracting this on a separate script. This cause the caller script `autoMigrate.sh` to exit immediately.
I'll fix this and run some more tests.